README.md

```diff
@@ -1,2 +1,96 @@
-# merge-sdk-kotlin
-The Kotlin/JVM SDK for accessing various Merge Unified APIs
+# Merge-SDK-Kotlin
+
+The Kotlin/JVM SDK for accessing various Merge Unified APIs. We use the following dependencies:
+
+* Ktor for http communication
+  * OkHttp engine (labeled Apache in ktor)
+  * Jackson serialization engine
+    * JSR 310 (java.time.OffsetDateTime for datetimes)
+* JUnit for tests
+* NO logging, aside from some `println` in tests
+
+We target Java language level 8 and above. The API methods are written using Kotlin coroutines, but can be called with
+`CompletableFutures` see [https://stackoverflow.com/a/52887677](this example). There should also be some packages that
+can make it work in Java 7, although we do not officially support it.
+
+## Usage
+
+For all examples, you can refer to the [src/test/kotlin/dev/merge/client/BasicTest.kt](BasicTest class) in this
+repository.
+
+### Plain call, Kotlin
+
+```kotlin
+val accountsApi = AccountsApi()
+accountsApi.setApiKey("REDACTED")
+accountsApi.setAccountToken("REDACTED")
+
+// or any other filters you prefer
+val accountingAccountsResponse = accountsApi.accountsList(AccountsApi.AccountsListRequest(
+    createdAfter=java.time.OffsetDateTime.now()
+))
+
+assertNotNull(accountingAccountsResponse)
+assertNotNull(accountingAccountsResponse.results)
+```
+
+### Expands call, Kotlin
+
+The `expand` parameter of Merge API endpoints take in properties which the caller wishes to "expand" from an id into
+a full sub object. For example, you can expand the applications property of a candidate from a list of ids of that
```

Comment:
full sub object? i don't think we use that terminology per https://help.merge.dev/en/articles/6153657-expand-parameter